src/client/index.ts

```diff
@@ -103,6 +109,17 @@ export class FortunaClient {
     })
   }
 
+  public async getTokenHoldersCount(
+    options: IGetHoldersCount
+  ): Promise<AxiosResponse<IHoldersCount[]>>
+  public async getTokenHoldersCount({ network, contracts }: IGetHoldersCount) {
+    const apiString = this.getApiUrl(network)
+
+    return this.axiosClient.post<IResponse>(`${apiString}/holders`, {
```

Comment:
why async if not using await?

Reply:
Michael did it with the other client methods, so I wanted to keep it the same. If I had to venture a guess it could be for documentation purposes. Like it's easy to look at it if it's asynchronous and know right away.

Reply:
interesting. not sure i love it this way tho.

else you can always just set a return type of a promise<>